src/coreclr/gc/env/common.h

```diff
@@ -21,6 +21,7 @@
 #include <assert.h>
 #include <stdarg.h>
 #include <memory.h>
+#include <math.h>
```

Comment:
which change requires including this header?

Reply:
fabsf